reddit-client/src/store/auth.js

```diff
@@ -8,6 +8,9 @@ const state = {
 const getters ={
```

Comment:
You can see this in the code I updated, but instead of creating getters that simply return a value from state, you can use `mapState` in the component.

Use getters for more complex logic, similar to computed properties.